githubtraining/training-manual
# Teaching scripts: fix issues with org owners

In the internal version of these scripts, org owners didn't get a copy of the repo. In the transition to the manual, somehow org owners get their own copy of the class repos as well.

So, this script achieves two things:
✅ ignore org owners so everyone that's an owner of the training org won't get a repo
✅ gives the facilitator a conflict-practice repo

To test:
- run `script/teach-class`
- run option 0 if you haven't used the scripts in this repo before
- run option 1 (create a caption this repo)
- ask someone that's not an org admin to comment on issue 1
- run option 2 (add collabs)
- run option 3 (create conflict repos)
- ensure that (1) student received a conflict repo, (2) facilitator received a conflict repo, (3) other org owners did not receive any conflict repos

@githubtraining/trainers could I get some 👀? This change is minimal and I'd like to 🚢 it quickly so at least 1 👍 would be ✨, perhaps @a-a-ron @brianamarie @hollenberry?
